edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/client/DeploymentKubernetesClientExtensions.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.Client
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using k8s;
+    using k8s.Models;
+
+    public static class DeploymentKubernetesClientExtensions
+    {
+        public static async Task AddModuleDeploymentAsync(this KubernetesClient client, string name, IDictionary<string, string> labels, IDictionary<string, string> annotations)
```

Comment:
since this is an extension method, should the method name show its purpose for testing only?

Reply:
KubernetesClient is a class used for our testing project, why not just adding these method to that class?